orca-queue/orca-queue.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
   compile "org.threeten:threeten-extra:1.0"
   compile "org.funktionale:funktionale-partials:1.1"
 
+  testCompile "org.springframework.boot:spring-boot-test:1.5.4.RELEASE"
```

Comment:
@cfieber does this seem like the right way to pull this dependency in? Do we have a constant for the Spring version somewhere? I'm wary of this getting out of sync.